src/js/angular/backup-and-restore/controllers.js

```diff
@@ -0,0 +1,65 @@
+const modules = [];
+
+angular
+    .module('graphdb.framework.monitoring.backupandrestore.controllers', modules)
+    .controller('BackupAndRestoreCtrl', BackupAndRestoreCtrl);
+
+BackupAndRestoreCtrl.$inject = ['$scope', '$interval', 'MonitoringRestService'];
+
+function BackupAndRestoreCtrl($scope, $interval, MonitoringRestService) {
+
+    $scope.loading = false;
+    $scope.initialized = false;
+
+    /**
+     * @type {BackupAndRestoreInfo[]}
+     */
+    $scope.backupAndRestoreInfos = undefined;
+    $scope.hasClusterOperation = false;
+    let timer = undefined;
+    const UPDATE_BACKUP_AND_RESTORE_INFO_DATA_TIME_INTERVAL = 2000;
```

Comment:
I would rather have this on top for easy access.

Reply:
done